lib/justifi.rb

```diff
@@ -66,6 +66,13 @@ def token
     def get_idempotency_key
       SecureRandom.uuid
     end
+
+    def with_seller_account(seller_account_id)
+      return unless block_given?
+      headers = {seller_account: seller_account_id}
+
+      yield headers
+    end
```

Comment:
@pete2786  @jnwaletzko That's almost how you guys described on your comments without changing every API resource we already have